pkg/api/router.go

```diff
@@ -90,6 +90,12 @@ func (s *server) setupRouting() {
 		),
 	})
 
+	handle(router, "/pss/send/{topic}/{targets}/{recipient}", jsonhttp.MethodHandler{
```

Comment:
why not reuse the same handler and just define the recipient mux variable as optional?